src/main/webapp/assets/modules/post-contest/postContestRewardDirective.js

```diff
@@ -0,0 +1,10 @@
+techlooper.directive("postContestReward", function ($http, utils) {
+  return {
+    restrict: "E",
+    replace: true,
+    templateUrl: "modules/post-contest/postContestReward.html",
+    link: function (scope, element, attr, ctrl) {
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.<br>'scope' is defined but never used.